vpr/src/base/setup_vpr.cpp

```diff
@@ -363,20 +362,20 @@ static void setup_timing(const t_options& Options, const bool TimingEnabled, t_t
  * @brief This loads up VPR's arch_switch_inf data by combining the switches
```

Comment:
I think we should move this doxygen comment to the declaration of the function near the top of the file.